src/NServiceBus.PersistenceTests/IPersistenceTestsConfiguration.cs

```diff
@@ -47,9 +47,9 @@ public interface IPersistenceTestsConfiguration
     // Consumers of this source package have to implement the remaining properties via partial class to configure the tests infrastructure.
     public partial class PersistenceTestsConfiguration : IPersistenceTestsConfiguration
     {
-        public PersistenceTestsConfiguration(TestVariant variant, TimeSpan? sessionTimeout = null)
+        public PersistenceTestsConfiguration(TestVariant variant)
         {
-            SessionTimeout = sessionTimeout;
+            SessionTimeout = variant.SessionTimeout;
```

Comment:
could we also completely remove the SessionTimout property here and just use `Variant.SessionTimeout` instead (and assign it a default value that can be overridden)?

Reply:
I think we could do that. But assigning a default value might be not a good idea. It seems the persisters check for it being not set

https://github.com/search?q=org%3AParticular+SessionTimeout&type=code

Reply:
With the change I applied, it would remain more or less compliant with how it behaved before.